plugins/org.jkiss.dbeaver.ui.editors.connection/src/org/jkiss/dbeaver/ui/internal/UIConnectionMessages.properties

```diff
@@ -28,8 +28,8 @@ controls_client_homes_panel_label_product_name = Product Name
 controls_client_homes_panel_label_product_version = Product Version
 controls_client_homes_panel_link_message = See <a>how to configure local clients</a> in our wiki
 
-dialog_setting_connection_driver_properties_description = Driver properties
-dialog_setting_connection_driver_properties_title = Driver properties
+dialog_setting_connection_driver_properties_description = Here are the default values for all connections. These will be applied if the properties are not defined in the Driver properties within the Connection settings.
+dialog_setting_connection_driver_properties_title = Default properties
```

Comment:
It looks like not what we wanted.
Default properties are on driver edit dialog. In the connection edit dialog we want to keep everything as is.